### PR TITLE
Rename Uri to URI

### PR DESCRIPTION
**Pull request by MalloZup**
Uri --> URI
add some basic comments for exported function(fixing golint)

### CODE REVIEW

File: libvirt/provider.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 func Provider() terraform.ResourceProvider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
-			"uri": &schema.Schema{
+			"URI": &schema.Schema{
```
Comment (flavio) [inline libvirt/provider.go]: Let's leave that to `uri` otherwise all the existing terraform files will be broken.

File: libvirt/provider.go [hunk 2]
```diff
@@ -30,7 +30,7 @@ func Provider() terraform.ResourceProvider {
 
 func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 	config := Config{
-		Uri: d.Get("uri").(string),
+		URI: d.Get("URI").(string),
```
Comment (flavio) [inline libvirt/provider.go]: Same here, let's leave that to `uri` otherwise all the existing terraform files will be broken.

**Comment by MalloZup** [comment]
ok thx i will do it :running_man:  :rofl:

**Comment by MalloZup** [comment]
thx

**Pull request merged**